fastlane/Fastfile

```diff
@@ -18,6 +18,10 @@ platform :ios do
     )
     slack(message: "New nightly build uploaded to iTunes Connect")
   end
+  desc "Remove inactive TestFlight users"
+  lane :clean do
+    clean_testflight_testers
```

Comment:
The only thing I have found is that if you want to run this script and get more than 100 people removed from the list, you need to include 
`clean_testflight_tester(days_of_inactivity: 30)`
to be really effective (often around 600-800 get removed at that value it seems).

Reply:
30 is the default now, so we are all good :)